clang/tools/libclang/CXIndexDataConsumer.cpp

```diff
@@ -560,7 +562,7 @@ bool CXIndexDataConsumer::handleDecl(const NamedDecl *D,
 
   if (shouldSuppressRefs())
     markEntityOccurrenceInFile(D, Loc);
-  
+
```

Comment:
Every other change in this file is unrelated to the linkage enum and should be backed out (committed separately).